hugr-core/src/extension/prelude.rs

```diff
@@ -753,7 +753,7 @@ impl MakeExtensionOp for UnpackTuple {
     }
 
     fn type_args(&self) -> Vec<TypeArg> {
-        vec![TypeArg::Sequence {
+        vec![TypeArg::List {
```

Comment:
I find it quite surprising that panic, exit, (un)packTuple take `TypeArg::List` instead of `TypeArg::Tuple`. I guess it's specifically `TypeArg::List(TypeTypeArg)` so that all the args are types

Reply:
Yes, that's how I understood the intention here.